actors/common/src/main/java/org/sunbird/metrics/actors/CourseMetricsActor.java

```diff
@@ -138,7 +127,9 @@ private void courseProgressMetricsV2(Request actorMessage) {
 
     Map<String, Object> result =
         ElasticSearchUtil.complexSearch(
-            searchDTO, ProjectUtil.EsIndex.sunbird.getIndexName(), EsType.user.getTypeName());
+            searchDTO,
+            ProjectUtil.EsIndex.sunbird.getIndexName(),
```

Comment:
is coursebatch stats is stored in "searchindex" index, because in last release for each type we have different-2 index